packages/web/src/pages/chat-page/components/ChatComposeModal.tsx

```diff
@@ -0,0 +1,107 @@
+import { useCallback, useState } from 'react'
+
+import {
+  chatActions,
+  ID,
+  searchUserListActions,
+  searchUserListSelectors,
+  SEARCH_USER_LIST_TAG,
+  userListActions
+} from '@audius/common'
+import {
+  Button,
+  ButtonSize,
+  ButtonType,
+  IconMail,
+  Modal,
+  ModalHeader,
+  ModalTitle,
+  Scrollbar
+} from '@audius/stems'
+import { useDispatch } from 'react-redux'
+
+import { useModalState } from 'common/hooks/useModalState'
+import Input from 'components/data-entry/Input'
+import ConnectedUserList from 'components/user-list/UserList'
+
+import styles from './ChatComposeModal.module.css'
+
+const { getUserList } = searchUserListSelectors
+const { createChat } = chatActions
+
+const messages = {
+  title: 'New Message',
+  searchUsers: 'Search Users',
+  message: 'Message'
+}
+const CHAT_COMPOSE_TAG = 'CHAT_COMPOSE'
+
+export const ChatComposeModal = () => {
+  const dispatch = useDispatch()
+  const [isVisible, setIsVisible] = useModalState('ChatCompose')
+  const [pendingUserChat, setPendingUserChat] = useState<number>()
+  const [query, setQuery] = useState('')
+
+  const doSearch = useCallback(() => {
+    dispatch(searchUserListActions.setSearchQuery(query))
+    dispatch(userListActions.reset(SEARCH_USER_LIST_TAG))
+    dispatch(userListActions.loadMore(SEARCH_USER_LIST_TAG))
```

Comment:
Seems like the connected user list component should do this itself on initial render, but it wasn't working for me. Also I think this use case is different in that there's not really a need to paginate and keep loading more, but rather to keep requerying and replacing the existing content.

Wondering if it makes more sense to not use the connected version, but just the presentation version with custom logic? Or even ditch the idea of a user list modal for this case but render something similar, but different...